webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -3167,8 +3166,8 @@ def test_if_user_b_has_no_elster_account_then_set_correct_info(self):
         self.assertEqual(expected_information, step.render_info.additional_info['dependent_notes'])
 
     def test_if_user_wants_no_cheaper_check_then_set_correct_info(self):
-        expected_information = ['form.eligibility.result-note.deadline',
-                                'form.eligibility.result-note.capital_investment']
+        expected_information = [_('form.eligibility.result-note.capital_investment'),
+                                _('form.eligibility.result-note.deadline')]
```

Comment:
I don't think this should be necessary as the `_` function is mocked for the function call beneath?

Reply:
Mocked the _l functions